flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskUpdateImports.java

```diff
@@ -222,7 +222,11 @@ private class UpdateFallBackImportsFile extends AbstractUpdateImports {
 
         UpdateFallBackImportsFile(ClassFinder classFinder, Options options) {
             super(options);
-            generatedFallBack = new File(options.getGeneratedFolder(),
+            generatedFallBack = new File(
+                    FrontendUtils
+                            .getFlowGeneratedImports(
+                                    options.getFrontendDirectory())
+                            .getParentFile(),
                     FrontendUtils.FALLBACK_IMPORTS_NAME);
```

Comment:
```suggestion
                    FrontendUtils
                            .getFlowGeneratedFolder(
                                    options.getFrontendDirectory()),
                    FrontendUtils.FALLBACK_IMPORTS_NAME);
```
Lets use the simple folder instead of the one with the extra file name

Reply:
Fixed